exoctk/utils.py

```diff
@@ -33,6 +29,36 @@
 # Set the version
 VERSION = '0.2'
 
+# Get the location of EXOCTK_DATA environvment variable and check that it is valid
+EXOCTK_DATA = os.environ.get('EXOCTK_DATA')
+
+# If the variable is blank or doesn't exist
+ON_TRAVIS = os.path.expanduser('~') == '/home/travis' or os.path.expanduser('~') == '/Users/travis'
+if not ON_TRAVIS:
+    if not EXOCTK_DATA:
+        raise ValueError(
+            'The $EXOCTK_DATA environment variable is not set.  Please set the '
+            'value of this variable to point to the location of the ExoCTK data '
+            'download folder.  Users may retreive this folder by clicking the '
+            '"ExoCTK Data Download" button on the ExoCTK website.'
+        )
+
+    # If the variable exists but doesn't point to a real location
+    if not os.path.exists(EXOCTK_DATA):
```

Comment:
I tried this with a `None` result if the path doesn't exist, and it fails with the error :
```TypeError: stat: path should be string, bytes, os.PathLike or integer, not NoneType```

If you like you could cut out the first check and run this one with some error catching.

Reply:
@julesfowler I'm having trouble understanding the situation you are describing. What exactly did you set for `$EXOCTK_DATA` in this situation? 

When I set it to a location that doesn't exists (e.g. `export EXOCTK_DATA=/foo/bar/`), I get the custom `FileNotFoundError` as expected.